DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyCheckBox.cs

```diff
@@ -53,7 +53,7 @@ public class GameLobbyCheckBox : XNAClientCheckBox
         /// The side index that this check box disallows when checked.
         /// Defaults to -1, which means none.
         /// </summary>
-        public int DisallowedSideIndex { get; set; } = -1;
+        public string[] DisallowedSideIndex;
```

Comment:
Change this to an `int` array and parse it as such when parsing the value of the key. Then the actual logic doesn't need to do any type conversions.

Reply:
thanks Rampa, I will update the PR accordingly